### PR TITLE
Crit rework minimal changes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Credit to Cogewerks and Goon for sprites and the base underlying idea of this system.

Alright, here's the crit rework redone.

I'd highly encourage you to read anything/everything so it's clear what has and has not changed. I'd also like to clear up some misconceptions about the intentions of the crit rework, as well. For those who are more interested in a quick and dirty of what has/has not changed, just jump to the bullet points below. **As there has been fairly dramatic alterations to the original PR, I highly recommend you read the bullet points, at least, before commenting.**

**I'd also encourage you to ask questions about how something functions, if you do not understand, as there was a lot of misunderstanding as to how things worked or how to treat in certain circumstances**

First off, the reason for the change to crit.

Current crit isn't particularly engaging; you inject a single medicine (epinephrine) then slap on a few patches/throw them into cryo and they're good to go in about 30-120 seconds (or a bit longer if they need surgery).

Likewise, for the player, your experience is largely “hit black screen and stare at it until dead”.

New crit solves both of these problems; it requires a bit more engagement on the doctors' behalf to treat  new symptoms and also forces them to not be so reliant on cryo. It also offers a bit more “wiggle room” for patients who are in crit. You can scream for help, you can try to make your way to medbay, or you have time to administer treatment to yourself. While there is still a black screen for players, eventually, it occurs much later, with this system, and at the point you would be **dead** with the old crit system; now, doctors have a chance to fight for your life.

Misconceptions I'd like to clear up:

The crit rework is absolutely not meant to be make death more meaningful, nor is it meant to push users into using any particular revive method; it's meant to be a way to alter how death, itself, and treating dying patients is handled. It offers both doctors and players a bit more wiggle room while still keeping medbay relevant and extremely important.

Things that **have changed** or been added in this crit rework:

- Crit goes through multiple stages, with more and more damage leading to accumulating negative status effects on you; these can be treated with various medications
- Three on-station species utilize the old crit system, or else they'd become unkillable (Diona, Slime, and IPC). 
  - You may have noticed Plasmamen weren't included here. They now utilize the new system with minimal changes to the species (the only change is they can now experience heart attacks)
- Other semi-playable species that use the old crit system: Skeletons, Abductors, Shadowlings, and Shadow People
- Medical Vendors have many additional medicines for treating patients in critical condition
- Cryo has been tuned downwards a bit, but it is still useful for keeping individuals in stasis
- Corazone's recipe has been removed. There's no two ways about this; this medication is completely overpowered with this new system.
- CPR has been dramatically buffed; it completely resets a patient's losebreath and dramatically heals O2 loss; it, together with epinephrine, can keep a shallow crit patient stable for a long time.
- There is a new “handheld defibrillator” . These smaller defibs do **NOT** revive, but are more effective for treating patients suffering full out cardiac arrest. They fit in medical belts and don't need to be recharged. Use these to treat; use the big defibs to bring back from the dead.
  - Any doctor's locker that has a defib will also spawn with one of these. Medical borgs also have one
- Bacon grease traitor reagent that induces immediate heart failure
- Tweaks to the upgraded robotic heart; it can stop both cardiac failure and cardiac arrest. It's also slightly less punishing if you get shocked while you have on (EMP's are still the bane of your existence, though).
- Tweaks the Health analyzer presentation, so it's easier to to see when someone is having cardiac arrest; see picture in a different post, below

Things that **haven't changed**:

- Advanced Trauma and Burn packs are still around and still in the vendors
- Strange Reagent is unchanged and functions as it currently does
- Cloning is unchanged and functions as it currently does
- Defibs are unchanged and function as they currently do, up to, and including 2 minute revivals
- You still have epipens in your survival box
- Surgery is unchanged; you still use advanced trauma packs for internal damage surgery
- There are no neurological kits in this PR; again, advanced trauma and burn kits remain

Now, for how to treat patients using this new system:

- If a patient has shock, you'll have to treat them with saline for it to go away; treating the underlying damage can help too.
- If a patient has cardiac failure, treating them epinephrine and/or atropine can remove cardiac failure
- If a patient has cardiac arrest, utilizing a handheld defib or a full defib can remove the heart attack; it's highly recommend to use the handheld defib for treating patients under cardiac arrest and not the full defibrillator; handheld defibs are specialized for treating cardiac arrest.
- Obviously, treat the underlying damage

How the new system works:

- When you lapse into critical condition you slowly accumulate O2 damage and roll for a chance to go into shock. The more damage you have, the higher a chance of you going into shock.
- When you have over 150 accumulated damage you can start rolling for a chance to acquire cardiac failure. This mostly gives you scary messages, but once it hits stage 3, it'll worsen the rate at which you acquire O2 damage and roll for a chance to put you into cardiac arrest. As with shock, the more damage you have, the higher a chance of acquire cardiac failure
- When you have over 200 damage (at this point, you'd be dead with the current implementation of crit), you roll for a chance to acquire a heart attack which makes you rapidly acquire O2 damage and brain damage.
-  At any point past 200 damage a patient can straight up die, though the chances are strongly weighted for brain damage.

Other Crap
- Fixed health analyzer sprite facing the wrong way
- Fixed some inconsistency with damage procs/direct var setting for damage
- Refactored death/crit thresholds to use a define
- Cleaned up health analyzer code ever so slightly

:cl: Fox McCloud
add: Crit has been dramatically overhauled. Treating patients now requires a broader range of medicine. Recommend checking out a guide for how to treat using the new system!
add: Medical vendors now have styptic and silver sulfadiazine, saline, atropine, mannitol, mutadone, calomel, diphenhydramine, oculine, insulin, morphine, and potassium iodide for helping treat new patients. Amount of Advanced Trauma and Burn packs has also been increased.
add: Adds new handheld defibs. Utilize these for treating patients with cardiac arrest. Note: these do not revive people from the dead!
add: Doctors lockers now have a handheld defib in addition to a regular defib.
add: Medical borgs now have a handheld defib
tweak: CPR dramatically buffed. Heavily lowers O2 damage and resets losebreath to 0
tweak: Cryoxadone tuned down a little
tweak: Health analyzer UI altered ever so slightly for cardiac arerest, so it's easier to find
add: Added bacon grease reagent to traitor poison bottles; induces immediate cardiac failure
tweak: Upgraded Robotic Hearts now attempt to correct both cardiac failure and cardiac arrest
tweak; Upgraded Robotic Hearts now do not punish you quite as severely for receiving a shock while having one (EMPs will still destroy you though)
del: Corazone recipe removed
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
_Round two..._
# FIGHT

**Comment by Spartan6** [comment]
Sprite for the 'handheld defib'?

**Comment by craftxbox** [comment]
whats the compact defib replaced with, then?

**Comment by MINIMAN10000** [comment]
> CPR has been dramatically buffed; it completely resets a patient's losebreath and dramatically heals O2 loss; it, together with epinephrine, can keep a shallow crit patient stable for a long time.

Oh cool, I was disappointed CPR seemed to have no practical use I'm glad to hear that it will have one now.

> There is a new “handheld defibrillator” . These smaller defibs do NOT revive, but are more effective for treating patients suffering full out cardiac arrest. They fit in medical belts and don't need to be recharged. Use these to treat; use the big defibs to bring back from the dead.

I enjoy item diversity. They have a use, are easy to carry, are more common, and don't need to be charged. Seems like a fine option to me. Although I don't know in practice how often "suffering full out cardiac arrest." So I can't say for certain how important it is.

> While there is still a black screen for players, eventually, it occurs much later, with this system, and at the point you would be dead with the old crit system; now, doctors have a chance to fight for your life.

I always felt once you went crit you just ceased being able to do anything was kinda lame, I've always personally hoped for a bit more of a fighting chance ( But I've never seriously played medical ) 

Currently it has always been "Dead but revivable? Defib and slap em in cryo" "Damaged? Slap em in cryo" "Dead but not revivable? Rev up the cloner" These choices are ultimately chosen due to a mix of ease of use and high effectiveness. So having "Crawling while crit" and "Handheld defib heart issues" sounds like it introduces a couple new mechanics.

Not everyone understands SR so typically no one bothers... in fact due to the fact SR is consumed even if no soul is present and has limited uses I find it very annoying. Honestly I wish both SR and Cloner would allow a person a like 5 seconds to choose to jump into their body before saying "No soul found"

> Cryo has been tuned downwards a bit, but it is still useful for keeping individuals in stasis

Hmm I can't actually see what in the code changed here or how it plays out but considering cryo tube is just a tube filled with medical liquids it makes sense for cryoxadone itself to mostly be used for keeping a person cryogenically encased. I look forward to seeing what changed.

>Tweaks to the upgraded robotic heart; it can stop both cardiac failure and cardiac arrest. It's also slightly less punishing if you get shocked while you have on (EMP's are still the bane of your existence, though).

Cool, a compelling reason to get a robotic heart and risk EMP death.

>Bacon grease traitor reagent that induces immediate heart failure

Does this mean a traitor with a syringegun could run around 1 shotting everyone with a non upgraded heart and exposed skin? Is cardiac failure a failure over time or is it immediate failure of the heart?

TL;DR Looks like more choices and more options

**Comment by KasparoVy** [comment]
Hopefully the bacon grease doesn't end up working like dragonsbreath or whatever, ashing people left right and center.

Stoked to give this a go

**Comment by Mitchs98** [comment]
I'm...actually willing to try this out this time around. Now that changes don't seem randomly tossed on and things are explained better and such. Not gonna say for sure whether or not I'm for or against it until I try it out with the new changes.

**Comment by granodd** [comment]
I'm curious whether or not we can make it to where Plasma Men can be defibbed through their suits now that they can get heart attacks, or if that will remain as to make treating them still a hellish matter, a bit more compounded with cardiac arrests.

**Comment by SkeletalElite** [comment]
My main problem now with this system, with its adjusments, is that going into shock is pretty much a death sentence unless you get to medbay. This isn't too bad, unless you're an antag and can't use the medbay because you're wanted. Because you need specialized chems, you have to carry a tremendous amount of healing items to be able to treat your own wounds once you enter shock, and this means any injury which incurs shock is going to be a death sentence for you as an antag most of the time. If this med system is going to work, I think most antags need to be given more effective tools at treating their own injuries. Perhaps some item for traitors and nukies like a box full of "shock auto injectors" thats a shock cure all for all stages. Vampire power to regenerate from shock, Changelings can do this already obviously. Wizard is one that I think has the highest impact, a wizard in shock (obviously lich doesnt apply because skeleton) is just going to die and theres not a lot they can do about it, unless theyre running wand belt or heal staff, meaning either one of those or a playing a non affected species is pretty much necessary for them, if they plan on surviving a long time.

On top of that the really long time's to death negatively impact  few antags. Terror spiders have to  keep biting critted people over and over again until they can web them and corpses won't be able to be turned into zombies until they've been in crit for 3 years. I'm not sure how the interaction works between face hugged people and the time until they burst but if that's not effected whatever. 

My final concern is that under this new system, the time it takes to treat a patient who is seriously wounded is increased tremendously. This means that if anything resembling a crisis rears its head, medbay will not be able to handle it. Every antagonist special ability, item, power, ect. in the game that causes a crisis of some sort has been balanced around "old crit." Treatment time for seriously injured patients is so much longer, that it's better to just send the seriously injured people to a line at the cloner and treat the people who are only a little injured. The more people the doctors have walking out the medbay doors the better, and treating people who are badly injured, doesnt have many people walking out the door, but, letting badly injured people just clone and doing as much quick fixing as possible, results in the most effecient time spent, and as a result you're more likely to achieve the goal of more people going out of medbay than into medbay at any given time. The thing that worked about the old system was that, treatment times were roughly equal and the only thing that really increased the amount of time spent in medbay was the number of surgeries required, and even in the old system we saw that it was often times easier to just clone away some of the game's worst injuries than to actually fix them. Making serious injuries worse, only worsens this problem. Medbay needs to be more interactive while simultaneously not taking any longer to heal people than it already did before, if that is the goal behind new crit.

It may be less terrible when you're actually in crit and more interesting for doctors(some would even disagree with that), but Making bad injuries worse only serves to make medbay less equipped to handle mass death/injury and making crit harder to get out of only makes the game less fun for antagonists. Under this system, it's not really doable to be able to prepare yourself to treat your own injuries. This system makes you far to reliant on the medbay, especially chemistry

**Comment by Kyep** [comment]
Under this system, how do you know someone's health status?
- Can you tell someone's status (normal, crit, shock, cardiac failure, heart attack, dead) by their sprite? Including the sprites on medical HUDs?
- By examining them with shift-click?
- By scanning them with a normal health analyser?
- By scanning them with an advanced health analyser?
- By scanning them in a medical diagnostic pod?
- By looking at the display when you have them in a medical sleeper?
- By placing them on an operating table hooked up to an operating computer?

How do you minimize the chances that players will become frustrated because other players, not realizing they need medical help, accidentally leave them for dead, or in crit, or mistake them for sleeping?

**Comment by SkeletalElite** [comment]
While not similar to what my previous comment was about kyep's comment made think that being able to see if someone is in shock and what shock state theyre in on the med hud would be a great QOL change if this system were to be implemented. Still dont like the system as a whole though

**Comment by Dinarzad** [comment]
There's a defib icon implemented recently for the health HUD, a little heart ekg monitor. Could be used (If it is not already) to indicate someone is having heart issues. I think it's also used for bodies in defib range, but if you're in crit you'll also have a flashing health bar on top, so a lot more eye catching and indicative of "This person is top priority"

From reading through the changes, this seems a lot more palatable, and looks like a good fusion of the old systems and the new.  Most of my own concerns about the system seem addressed in this version.

Only suggestion I could really make without experiencing a test merge, would be since Corazone is now considered too powerful, but would be a massive help.
Perhaps a bottle or pill bottle of some of it could be a possible Sol Trader item? Outside the scope of this particular PR for sure, but still something to consider.

**Comment by Fox-McCloud** [comment]
>Hmm I can't actually see what in the code changed here or how it plays out but considering cryo tube is just a tube filled with medical liquids it makes sense for cryoxadone itself to mostly be used for keeping a person cryogenically encased. I look forward to seeing what changed.

Cryo injects its healing chems less often and heals less O2 damage on its own.

>whats the compact defib replaced with, then?

Compact defib is still a revival defib; I didn't touch the regular defib units; the **handheld defib** is a completely new item that has completely different rules and is *purely* for treating patients undergoing cardiac arrest or who are in deep critical condition

>Hopefully the bacon grease doesn't end up working like dragonsbreath or whatever, ashing people left right and center.

Cardiac failure isn't an insta-lose situation. It gives you scary messages at first, sometimes causes O2 damage and if left untreated *then* leads to you flopping over and accumulating a ton of O2 and brain damage because you'll be under full cardiac arrest. You can treat the situation by injecting yourself (or someone suffering from it) with epinephrine and/or atropine (having both in the bloodstream at once doubles the chances of curing it).

>Under this system, how do you know someone's health status?

You can tell all these new conditions with a regular health analyzer; it will state the condition, what stage it's at and the recommended way to treat the ailments. Additionally, patients get unique messages for particular ailments. For instance, if a patient is telling you that their "arm really hurts", that's indicative that they're undergoing cardiac failure, and you need to administer epinephrine and/or atropine ASAP.

>Antag concerns

Most humanoid antags that can undergo newcrit already have ways of self treating, to a degree (as with before). Everyone has emergency boxes which have epinephrine in them, which can treat cardiac failure. Additionally, if you're able to heal yourself up, you can get rid of shock that way. The only thing you can't address is full out cardiac arrest--but cardiac arrest is kinda the *new* "hard crit" per se; you're not meant to recover from it on your own.

>Concerns about treatment time

Generally speaking, the system sounds more daunting than it actually is. It's designed to add a bit more spice and action to treating a patient without necessarily dramatically altering the fundamental underlying skills needed to treat a patient; once you know how to treat shock, cardiac failure, and cardiac arrest (and how our existing chemicals interact with them, in addition to the handheld defib), the amount of time to treat someone in crit isn't dramatically different. There will be a little bit more time in some circumstances, but keep in mind, most damage is burns and brute, and we have instantaneous patches and trauma/burn kits for these. 

If you use the old methodology of throw them in cryo or the sleeper and pump them full of chems, then yeah, it'll take forever; but that's not how you're supposed to treat using this new system; you're supposed to treat primarily with directly administered medicine. This rewards doctors who utilize the correct knowledge and directly **interact with the patient**. This isn't just more rewarding from a gameplay standpoint, but it encourages more interaction and involvement, which gives a bit more RP opportunities too "Help, doc, my arm is killing me! My heart feels like it's about to give out!".

**Comment by Agameofscones** [comment]
Regarding the mini-defib, perhaps it'd be better to have it as an AED?
image
Makes a bit more sense to me. Does it have a belt state?

**Comment by Fox-McCloud** [comment]
the handheld defib does not fit on belts.

**Comment by farie82** [comment]
Looks way better then the previous one. Im still sceptical due to the way crit people are handled. In the sense that they wont go down. But Im all for a few test merges

**Comment by SkeletalElite** [comment]
AED sounds cooler than handheld defib so it should still be that instead, even if its not on the belt.
And that does sound not too bad. Well have to see in test merge.

**Comment by Mitchs98** [comment]
> Cryo injects its healing chems less often and heals less O2 damage on its own.
How much less often, comparatively to how it is now and how it is on this PR?

**Comment by EvadableMoxie** [comment]
I'm concerned that being able to revive with defibs might defeat the purpose of the system. What's stopping you from just letting them die, healing up all the damage while they're dead, then defibbing them and bypassing crit entirely?

I guess we'll see when it's test merged.

Oh, and you can put the omni back in the CMO's hypospray if that wasn't reverted? Really not a fan of that change creating more busy work for medical at round start.

**Comment by TDSSS** [comment]
For terror spiders, maybe they could be given some kind of finishing attack on people in cardiac arrest that insta-kills them? Something like the spider biting off their head once they are helpless?

**Comment by Fox-McCloud** [comment]
> In the sense that they wont go down.

There is still a hard cutoff threshold for blacking out where they will go down, but still be alive.

>Oh, and you can put the omni back in the CMO's hypospray if that wasn't reverted? Really not a fan of that change creating more busy work for medical at round start.

Unchanged. CMO's hypo still has omnizine.

**Comment by AffectedArc07** [comment]
````
Tweka; Upgraded Robotic Hearts now do not punish you quite as severely for receiving a shock while having one (EMPs will still destroy you though)
```

# Tweka 

Might want to fix that

**Comment by taukausanake** [comment]
please don't fix that

**Comment by MrMagolor** [comment]
> For the most part, existing chems are balanced around this new system already

Gooncrit, goonchem
It's all a conspiracy
(I mean as a goon player I can't really complain)

**Comment by Mitchs98** [comment]
> For terror spiders, maybe they could be given some kind of finishing attack on people in cardiac arrest that insta-kills them? Something like the spider biting off their head once they are helpless?

Decapitation is a bit too far. It'd making reviving them harder than it needs to be if you manage to find their body. Especially if the spiders hide/space the heads(which you know they'll do..).

I also didn't get my question answered...sort of feel ignored.

**Comment by Landerlow** [comment]
I like these changes very much, as opposed to the previous ones. I'm interested as to how this will play out.

**Comment by Fox-McCloud** [comment]
>How much less often, comparatively to how it is now and how it is on this PR?

O2 healing from 10 to 6 and it injects every 17 cycles instead of 10. Doesn't have much effect on Cryoxadone use, but it does prevent "put 5 healing chems in the cryotube" non-sense.

**Comment by Fox-McCloud** [comment]
One last change to health analyzers, so the information is presented a little bit more clearly and what they're suffering from doesn't get lost.

img

File: appveyor.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-  - ps: "[System.IO.Compression.ZipFile]::CreateFromDirectory($env:APPVEYOR_BUILD_FOLDER + '/deploy', $env:APPVEYOR_BUILD_FOLDER + '/deploy.zip')"
-artifacts:
+version: '{build}'
+skip_branch_with_pr: true
+shallow_clone: true
+build_script:
+  - ps: ./tools/appveyor/build.ps1
+  - ps: "$deployPath = $env:APPVEYOR_BUILD_FOLDER + '/deploy'; bash ./tools/appveyor/deploy.sh $deployPath" 
+  - ps: "[System.IO.Compression.ZipFile]::CreateFromDirectory($env:APPVEYOR_BUILD_FOLDER + '/deploy', $env:APPVEYOR_BUILD_FOLDER + '/deploy.zip')"
+artifacts:
```
Comment (Fox-McCloud) [inline appveyor.yml]: @AffectedArc07 

What did you do, and why is this showing up in a merge from upstream commit.

**Comment by Pennwick** [comment]
Can you offer more details on the Upgraded Cybernetic heart? Its anti stun properties look VERY significant. Reducing stun times by 50% normally and 2/3rds when emagged. What are the odds of getting the various penalties when hit with a stun baton, both emagged and with safeties on.

Last test run cloning seemed inconsistent, sometimes people came out of cloning with serious heart attacks and died shortly after, other times they were 100% fine. Does this still happen or is it more consistent.

The medical vendor seems overstocked. Calomel, Pot Idoine, diphenhydramine, occuline? These aren't exactly staples. With all these chems in the 4 medical vendors there isn't a whole lot for chemists to make. Strange Reagent, Pentenic Acid, and hydrocodine are pretty much all that are left. Any reason that there are so many chems available at roundstart.

This is personal opinion but I liked the Fix-o-vein being whats used for organ patching. Both kind of make sense but it always bothered me that the trauma kits weren't depleted. Plus if its the fix-o-vein then the surgery implant has everything the surgeon needs.

**Comment by Fox-McCloud** [comment]
>Can you offer more details on the Upgraded Cybernetic heart? Its anti stun properties look VERY significant. Reducing stun times by 50% normally and 2/3rds when emagged. What are the odds of getting the various penalties when hit with a stun baton, both emagged and with safeties on.

I'd check out the cybernetic heart PR for descriptions of what it does; it's not really a point of discussion here aside from what it *additionally* does in relation to this PR.

It attempts to also cure cardiac failure (in addition to cardiac arrest); the chance to cure cardiac failure is a bit higher than cardiac arrest.

>Last test run cloning seemed inconsistent, sometimes people came out of cloning with serious heart attacks and died shortly after, other times they were 100% fine. Does this still happen or is it more consistent.

Are you sure about this? Cloners inject corazone, salbutamol, saline, and epinephrine. These address all medical conditions and out and out prevent cardiac arrest; they really shouldn't be popping out with any of these conditions.

>The medical vendor seems overstocked. Calomel, Pot Idoine, diphenhydramine, occuline? These aren't exactly staples. With all these chems in the 4 medical vendors there isn't a whole lot for chemists to make. Strange Reagent, Pentenic Acid, and hydrocodine are pretty much all that are left. Any reason that there are so many chems available at roundstart.

Mitocholide, synthflesh, synth meat, perfluorodecalin, antihol, and rezadone are all additional chems worth making, not to mention shoring up supply shortages if/when things get chaotic. There's also converting vendor supplies from one form to another, as well.

**Comment by Fox-McCloud** [comment]
Further clarification for the heart; having a missing heart or necrotic heart will state that the patient has that while not stating they have cardiac arrest. Despite the fact they all do the same thing, this is to avoid confusion.

Also did the same for brain damage; someone who's missing their brain isn't "brain dead", they just plain don't have a brain.

**Comment by Mitchs98** [comment]
Might I suggest for the scanner, since not everyone knows what 'Cardiac Stimulants' are, putting the medicine types there? It'll be a great boon to a newbie doctor and even an experienced one who is a bit rushed and overwhelmed. Just a minor QOL suggestion.

**Comment by Mitchs98** [comment]
@Fox-McCloud Just a heads up. The advanced brute/burn patches aren't actually in the vendors. I believe you forgot to edit that back in when opening the new PR, given the fact the PR states they're supposed to be in it. Was checking something else and noticed that.

**Comment by Fox-McCloud** [comment]
@Mitchs98 

You're just plain wrong

`/obj/item/stack/medical/bruise_pack/advanced = 4, /obj/item/stack/medical/ointment/advanced = 4`

**Comment by Mitchs98** [comment]
I did look. I however didn't notice the vendors now had both @Fox-McCloud  My bad on that part

**Comment by MrMagolor** [comment]
I was wondering why this hadn't been commented on in awhile- then again, it's probably for the best to delay the implementation for a while, _especially_ because of the influx of new players. ~~relist para when~~

**Comment by Spacemanspark** [comment]
While I'm not sure how I feel about this, I'd rather we either decided whether or not to merge this sooner rather than later, so we don't have the massive waves of newer players learning one system amongst the fifty billion we already have, and then have it radically change a month or so down the road when they're just starting to get a grip on things.

**Comment by Mitchs98** [comment]
I personally haven't gotten a chance to try it this time around, nor do I really intend to, due to the wave of players. Medical was already hellish enough without this implemented, this is only compounding the lack of new people knowledge making medbay hell now.

I honestly think this updated version of newcrit could be interesting, though the few experiences I have with it leads me to believe it's horribly unbalanced for a few things involving death. Most notably holoparasites get a _massive_ buff with this. One I don't feel is too needed. Though I'm sure there's ways to balance that, it's just something I noticed.

**Comment by MrMagolor** [comment]
> Most notably holoparasites get a massive buff with this. One I don't feel is too needed. Though I'm sure there's ways to balance that, it's just something I noticed.

Forgive me, but I don't know how holoparas get a big buff with this? AFAIK you can still just attack people to death with copious amounts of damage, same as it was before.

**Comment by Mitchs98** [comment]
> > Most notably holoparasites get a massive buff with this. One I don't feel is too needed. Though I'm sure there's ways to balance that, it's just something I noticed.
> 
> Forgive me, but I don't know how holoparas get a big buff with this? AFAIK you can still just attack people to death with copious amounts of damage, same as it was before.

The holopara stays alive until you die of braindeath. Which, is a MASSIVE buff in MANY situations.

**Comment by Fox-McCloud** [comment]
to do, before this is merged:

- re-add bandages to the vendor (oops, unintentionally removed them)
- alter the `InCritical` to not fail under certain circumstances

Aside from that, this is pretty much ready.

**Comment by farie82** [comment]
As a doc it's fine to work with. Just means you have to use more chemicals and more salbutanol. In other words have a pill bottle with you and a patch pack. And you have to use cloning more. 
Is it better? Eh not sure for me.

From an officers or antag perspective it's horrible. People will not go down or die due to it. You can have -700 HP and still live with a holoparasite in you. This is fixable by changing the holoparasite.
But having to kill people because you can't get them into hardcrit anymore is ... well not fun. It's based of off RNG so you will get cases where somebody just won't get a heart attack or such and will keep on fighting till he wins. And dies afterwards.

Due to this I still am not a fan of the PR in it's current form. It does make medbay different. More reliant on chemicals which are available in the medi vendors. I've yet had to rely on chemistry.

**Comment by Mitchs98** [comment]
> As a doc it's fine to work with. Just means you have to use more chemicals and more salbutanol. In other words have a pill bottle with you and a patch pack. And you have to use cloning more.
> Is it better? Eh not sure for me.
> 
> From an officers or antag perspective it's horrible. People will not go down or die due to it. You can have -700 HP and still live with a holoparasite in you. This is fixable by changing the holoparasite.
> But having to kill people because you can't get them into hardcrit anymore is ... well not fun. It's based of off RNG so you will get cases where somebody just won't get a heart attack or such and will keep on fighting till he wins. And dies afterwards.
> 
> Due to this I still am not a fan of the PR in it's current form. It does make medbay different. More reliant on chemicals which are available in the medi vendors. I've yet had to rely on chemistry.

Basically my views as well. Having to kill people to do things just isn't great. Having nigh immortal holopara's...also not great.

**Comment by KasparoVy** [comment]
Beheading meta

**Comment by Fox-McCloud** [comment]
>But having to kill people because you can't get them into hardcrit anymore

This is incorrect. 

At -100 health, they WILL become unconscious outside of very, very very specific circumstances; in 99% of engagements, at -100 health (200 damage), they will *always* lose consciousness.
